packages/confirm-dialog/src/vaadin-lit-confirm-dialog.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @license
+ * Copyright (c) 2018 - 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import '@vaadin/button/src/vaadin-button.js';
```

Comment:
I wonder if the lit component should import `vaadin-lit-button.js`? If you want to use Lit-based Vaadin components (including confirm dialog and button) on a page, then this would cause a conflict.

Reply:
Thanks, changed.